react/features/participants-pane/index.js

```diff
@@ -0,0 +1,6 @@
+export * from './actions';
```

Comment:
Drop this file altogether. Import specific things you need from each file / section. This avoids having weird import cycles that tend to break mobile.